src/helpers/issue.ts

```diff
@@ -658,6 +658,16 @@ export const getPullByNumber = async (context: Context, pull_number: number) =>
   }
 };
 
+export const getRequestedReviewerStart = async (user: string) => {
```

Comment:
```suggestion
export async function getRequestedReviewerStart (user: string) {
```

Changing all of these to named functions for stack traces.